test/modules/Modules.Test/TestResultCoordinator/Reports/DirectMethod/DirectMethodLongHaulReportData.cs

```diff
@@ -122,7 +122,7 @@ class DirectMethodLongHaulReportData
                         new DateTime(2020, 1, 1, 9, 10, 24, 15)
                     },
                     7,
-                    false,
+                    true,
```

Comment:
The tests should pass in this case. I am not sure how it passed checkin in the first place. Maybe tests don't run there? Need to confirm.

I didn't even change anything in the longhaul code path.